lstchain/scripts/lstchain_mc_dl2_to_irf.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+"""
+Script to generate IRFs from MC DL2 files, using pyirf functions.
+This will save the IRFs, and either use fixed cuts or optimise them.
+and later it can be used by lstchain_dl2_to_dl3 to generate DL3 files.
+
+TODO: Add other IRFs
+Use optimised cuts and binning
+
+- Input: Path where the merged DL2 data HDF5 files are present
+- Output: IRFs compiled in fits.gz format
+
+Usage:
+$> python lstchain_mc_dl2_to_irf.py
+--input-file-gamma ./gamma/dl2_gamma_*.h5
+--output-dir ./IRFs/
```

Comment:
The option is output-irf-dir
```suggestion
--output-irf-dir ./IRFs/
```